docs/index.rst

```diff
@@ -115,8 +115,8 @@ Mission Statement
 
     zreferences
 
-====
+-----------------
 News
-====
+-----------------
```

Comment:
No. of characters in under/above lines should be exactly same as that of the text enclosed. Please make them 4.